tests/test_util.py

```diff
@@ -0,0 +1,125 @@
+''' Unit tests for boss.util module. '''
+
+import pytest
+
+from boss.util import (
+    halt,
+    info,
+    echo,
+    merge,
+    is_iterable
+)
+
+def test_halt():
+    ''' Test for boss.util.halt() '''
+    message = 'Test message'
+
+    with pytest.raises(SystemExit, match=message):
+        halt(message)
+
+def test_echo(capsys):
```

Comment:
Inconsistent blank lines here, as [pep8](https://www.python.org/dev/peps/pep-0008/#blank-lines) requires 2 blank lines in between function definitions. You can use something like [this](https://github.com/hhatto/autopep8) too.

Reply:
Are you using an `autopep8` command to format the code? It'll be helpful to add that to `t`.

Reply:
@srishanbhattarai Actually, I'm using a vscode plugin right now. But, I think adding a command to `t` would be pretty handy too.